fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/home/HomeActivity.java

```diff
@@ -262,6 +264,16 @@ public void setupSocialMediaButton() {
     socialMediaButton.setOnClickListener(v -> mViewModel.openSocialMedia());
   }
 
+  public void setupDigitalCashButton() {
+    Button digitalCashButton = findViewById(R.id.tab_digital_cash);
+    digitalCashButton.setOnClickListener(
+        new View.OnClickListener() {
+          public void onClick(View v) {
+            Toast.makeText(HomeActivity.this, "Button for Transaction", Toast.LENGTH_LONG).show();
```

Comment:
In Android, we try to use string.xml to store any text that will be displayed to the user.

This makes localization (translation) very easy.